client/data/blogging-prompt/is-bloganuary.ts

```diff
@@ -4,5 +4,10 @@ import moment from 'moment';
  * @returns true if bloganuary mode is active
  */
 export default function isBloganuary() {
+	// Disable for January 2025 (see https://wp.me/p5uIfZ-gxX).
+	const BLOGANUARY_ENABLED = false;
+	if ( ! BLOGANUARY_ENABLED ) {
+		return false;
+	}
```

Comment:
Can we just use `return false` here? ..... alternatively we can use `moment` and if the year is `2025` then `return false`.

Reply:
I could `return false`, but then I'd need to rip out most of the code here (which the issue asked that we not do). I wanted to make it clearer that this was intentional and preserve the current return logic.

We could check for 2025, but I'm worried we'll forget to disable it for 2026. 😂